docker/images/iron/Dockerfile

```diff
@@ -44,6 +44,8 @@ RUN apt-get update \
     gdb \
     git \
     lcov \
+    libboost-all-dev \
+    libopenvdb-dev \
```

Comment:
@pvela2017 consider listing `openvdb` among `package.xml` dependencies. At least for ROS 2 builds it makes sense to pull it.

Reply:
Regarding that, when the `dockerfiles` use `rosdep install` to install the dependencies is it going to be ok for systems earlier than Noble (since the openvdb package is broken for those distros) or if it can install it regardless if the package is ok there will be no problem?